backend/infrahub/graphql/types/context.py

```diff
@@ -0,0 +1,5 @@
+from graphene import InputObjectType, String
+
+
+class ContextInput(InputObjectType):
+    account_id = String(required=False)
```

Comment:
I think it would be good to have a structure for account with `id` and `kind`
in the future I'm sure we'll add more data to the context object and it would make sense to not have them all at the top level

Reply:
I'll add the account an an object with ID as params. I'm a bit conflicted about the "kind" with regards to us having to validate that the account exists regardless and possibly having to overwrite the kind. Alternatively the mutation would simply fail if the account_id and kind combination is incorrect. I.e. by using the kind as a search for the NodeManager query. But then we'd have to validate that the object is inheriting from the CoreGenericAccount.

Reply:
Good point, let's not add it then